app/controllers/keyword_type.js

```diff
@@ -0,0 +1,82 @@
+"use strict";
+
+const KeywordType = require('../models/keyword_type');
+
+/**
+ * Get all KeywordTypes in the database
+ * @param  {Express.Request}   req  - the request object
+ * @param  {Express.Response}  res  - the response object
+ * @param  {Function} next - pass to next route handler
+ */
+function getKeywordTypes(req, res, next) {
+    return KeywordType.fetchAll()
+    .then(collection => res.json(collection.serialize()))
```

Comment:
Need to respond with a status code as well. This should be a 200. See the user controller for how to structure this.
Aside from that, the .json part looks good.